Labs/Lab_6/src/main/java/com/fmi/web/lab/raceeventmanagement/controller/EventController.java

```diff
@@ -0,0 +1,128 @@
+package com.fmi.web.lab.raceeventmanagement.controller;
+
+import com.fmi.web.lab.raceeventmanagement.model.Racer;
+import com.fmi.web.lab.raceeventmanagement.model.Team;
+import com.fmi.web.lab.raceeventmanagement.model.Track;
+import com.fmi.web.lab.raceeventmanagement.repository.sequence.TeamSequence;
+import com.fmi.web.lab.raceeventmanagement.service.EventService;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@RestController
+@RequestMapping("/controller")
+public class EventController {
```

Comment:
You don't need to convert everything to String in order for the client to understand the result. Use DTOs, which will be converted to JSON